src/core/config/mle.h

```diff
@@ -240,4 +240,14 @@
 #define OPENTHREAD_CONFIG_MLE_INFORM_PREVIOUS_PARENT_ON_REATTACH 0
 #endif
 
+/**
+ * @def OPENTHREAD_CONFIG_LINK_METRICS_ENABLE
+ *
+ * Define as 1 to enable Link Metrics feature.
+ *
+ */
+#ifndef OPENTHREAD_CONFIG_LINK_METRICS_ENABLE
+#define OPENTHREAD_CONFIG_LINK_METRICS_ENABLE 0
```

Comment:
```suggestion
#define OPENTHREAD_CONFIG_MLE_LINK_METRICS_ENABLE 0
```
As it is MLE config like other defines.